### PR TITLE
added call for sponsors

### PR DESCRIPTION
**Pull request by nceder**
Added text for call sponsors (based on last year's) 

Let me know if I should have done anything differently.

### CODE REVIEW

File: content/news/call-for-sponsors.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
++++
+title = 'Call for Sponsors'
+date = 2024-08-16T20:26:21-05:00
+draft = true
```
Comment (Mariatta) [inline content/news/call-for-sponsors.md:4]:
```suggestion
draft = false
```

Comment (Mariatta) [reply content/news/call-for-sponsors.md:4]: It doesn't seem to show up on the preview when the draft is set to True

Comment (nceder) [reply content/news/call-for-sponsors.md:4]: I guess that makes sense... I didn't know if it was to be set to true or false from the start or not. Now I guess I do. :-)

File: content/news/call-for-sponsors.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+date = 2024-08-16T20:26:21-05:00
+draft = false
+meta_title = "Help support this year's Global PyLadies Conference"
+description = "Help support this year's Global PyLadies Conference"
+tags = ["python", "community", "conference", "sponsorship", "pyladies"]
+layout = "news"
+categories = ["Blog Post",]
+image = ''
+imagealt = ''
++++
```
Comment (Mariatta) [inline content/news/call-for-sponsors.md:12]:
```suggestion
author: "Naomi" 
+++
```

I realized the `author` info is missing from the news front matter :( It's my mistake, I will fix it tomorrow but for now please add the author info accordingly.
Thank you.

Comment (nceder) [reply content/news/call-for-sponsors.md:12]: Okay, will do.

**Review by Mariatta — APPROVED** [review]
Thank you!

**Pull request merged**